WooCommerce/WooCommerceUITests/Tests/PaymentsTests.swift

```diff
@@ -19,4 +19,11 @@ final class PaymentsTests: XCTestCase {
             .tapChipperManual()
             .verifyChipperManualLoadedOnWebView()
     }
+
+    func test_load_learn_more_link() throws {
+        try TabNavComponent().goToMenuScreen()
+            .goToPaymentsScreen()
```

Comment:
Now, once there are two tests in class, this part can be moved to setUp. (same for `test_load_chipper_card_reader_manual`)

Reply:
good call! updated in 5cc61e70317c0ecfd1c1ea73ddeeebd93d55b740